docs/conf.py

```diff
@@ -424,6 +425,7 @@ def _get_rst_filepath_from_path(filepath: str):
     '*/airflow/kubernetes/kubernetes_request_factory/*',
     '*/_internal*',
     '*/node_modules/*',
+    '*/example_dags/*,',
```

Comment:
For unknown reasons, a new warning started popping up, so I had to fix it as well.
See: https://github.com/apache/airflow/runs/1367944822

Reply:
It does not stop to amaze me how unrelated changes trigger new warnings in sphinx autoapi. I have no trust in that software :)